crates/lib/src/qpu/mod.rs

```diff
@@ -27,7 +27,7 @@ pub(crate) use execution::{Error as ExecutionError, Execution};
 /// 1. Problem communicating with QCS
 /// 2. Unauthenticated
 /// 3. Expired token
-pub(crate) async fn get_isa(
+pub async fn get_isa(
```

Comment:
I'm a little hesitant about this pattern of lightly wrapping client calls in SDK calls with dubious names (while completely acknowledging that you didn't do that). It doesn't feel like something that's going to stand up well over time.

If nothing else, I think I'd be happier with a full name, `get_instruction_set_architecture` or `fetch_instruction_set_architecture`